packages/ui/react-ui/src/CrossmintStatusButton.tsx

```diff
@@ -14,22 +20,33 @@ export const CrossmintStatusButton: FC<BaseButtonProps> = ({
     clientId,
     auctionId,
     development = false,
+    platformId,
     ...props
 }) => {
     const status = useCrossmintStatus({ clientId, development });
 
+    const formatOnboardingQueryParams = () => {
+        const onboardingQueryParams: OnboardingQueryParams = {
+            clientId: clientId,
+        };
+
+        if (platformId) onboardingQueryParams.platformId = platformId;
+        if (auctionId) onboardingQueryParams.auctionId = auctionId;
```

Comment:
You're not passing nor using the mintConfig though right? Am I missing something? :S